opium/src/middlewares/middleware_router.mli

```diff
@@ -1,7 +1,7 @@
-type 'a t
+type t
 
-val m : Rock.Handler.t t -> Rock.Middleware.t
-val empty : 'action t
-val add : 'a t -> route:Route.t -> meth:Method.t -> action:'a -> 'a t
+val m : t -> Rock.Middleware.t
```

Comment:
Do you have any guidelines on naming conventions for similar functions in Opium? I assume `m` is "middleware", but would be good to know if this should be used more generally.

Reply:
Yeah, that's my original naming convention for a module that provides a middleware. It seemed a little long to name things like `Middleware_foo.middleware`. Not so sure if this convention is still beneficial.